lib/internal/options.js

```diff
@@ -1,15 +1,14 @@
-// https://github.com/nodejs/node/blob/54819f08e0c469528901d81a9cee546ea518a5c3/lib/internal/options.js
+// https://github.com/nodejs/node/blob/1aab13cad9c800f4121c1d35b554b78c1b17bdbd/lib/internal/options.js
 
 'use strict'
 
-const minimist = require('minimist')
-
-const argv = minimist(process.argv.slice(2))
+const argv = Object.create(null)
 
 function getOptionValue (optionName) {
   return argv[optionName.slice(2)] // remove leading --
```

Comment:
this always returns `undefined` now, right?

Reply:
Nope, the `argv` object is populated iif in the `bin/node--test.js` script.